src/main/java/com/psk/pyragas/ToolRent/test/TestPersistence.java

```diff
@@ -0,0 +1,32 @@
+package com.psk.pyragas.ToolRent.test;
+import com.psk.pyragas.ToolRent.dao.*;
+import com.psk.pyragas.ToolRent.entities.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.transaction.Transactional;
+
```

Comment:
Is this file to be deleted in future? (from name "Test")
If yes should files like these be in develop branch?

Reply:
Yeah, it doesn't work as a test class. I'll delete it.